contracts/libraries/NFTDescriptor.sol

```diff
@@ -110,22 +112,36 @@ library NFTDescriptor {
             );
     }
 
+    function formatTokenSymbol(string memory symbol) internal pure returns (string memory) {
+        bytes memory symbolBytes = bytes(symbol);
+        uint8 quotesCount = 0;
+        for (uint8 i = 0; i < symbolBytes.length; i++) {
+            if (symbolBytes[i] == '"') {
+                quotesCount++;
+            }
+        }
+        if (quotesCount > 0) {
+            bytes memory formattedBytes = new bytes(symbolBytes.length + (quotesCount));
+            uint256 index;
+            for (uint8 i = 0; i < symbolBytes.length; i++) {
+                if (symbolBytes[i] == '"') {
+                    formattedBytes[index++] = '\\';
+                }
+                formattedBytes[index++] = symbolBytes[i];
+            }
+            return string(formattedBytes);
+        }
+        return symbol;
+    }
+
     struct DecimalStringParams {
-        // significant figures of decimal
```

Comment:
prettier's gone rogue...

Reply:
Ugh, I'm so sorry about this. I just opened a PR that fixes this: https://github.com/prettier-solidity/prettier-plugin-solidity/pull/480 I'll try to have it merged and released today.

Reply:
Wow, Franco, that was so quick!! thank you! cc @fvictorio

Reply:
It should be fixed in `prettier-plugin-solidity@1.0.0-beta.10`. Let us know if it's not!

Reply:
yess, that did the trick 💯 happy to be able to throw you weird edgecases as you guys get closer to a stable release 😂